app/models/reference.rb

```diff
@@ -3,7 +3,7 @@ class Reference < ApplicationRecord
   validates :email_address, presence: true,
                             email_address: true,
                             length: { maximum: 100 },
-                            uniqueness: { scope: :application_form_id }
+                            uniqueness: { case_sensitive: false, scope: :application_form_id }
```

Comment:
This seems unrelated to the other changes, and doesn't come with a corresponding test, but I think it's fine to sneak it in